src/modules/Dashboard/components/TakeNowModal/LabeledDropdown/LabeledUserDropdown.tsx

```diff
@@ -64,42 +71,101 @@ export const LabeledUserDropdown = ({
-    <StyledFlexColumn sx={{ gap: 1.6, marginTop: 2.4 }}>
+    <StyledFlexColumn sx={{ gap: 1.6, ...sx }}>
       <Box sx={{ display: 'flex', gap: 0.4 }}>
-        <StyledTitleMedium fontWeight="bold">{label}</StyledTitleMedium>
-        <Tooltip tooltipTitle={tooltip}>
-          <Box sx={{ height: 24 }}>
-            <StyledTitleTooltipIcon
-              sx={{ marginLeft: 0 }}
-              id="more-info-outlined"
-              width={24}
-              height={24}
-              data-testid={`${rest['data-testid']}-tooltip-icon`}
-            />
-          </Box>
-        </Tooltip>
+        <StyledTitleMedium sx={{ fontWeight: 700, color: variables.palette.on_surface }}>
+          {label}
+        </StyledTitleMedium>
+        <RenderIf condition={!!tooltip}>
+          <Tooltip tooltipTitle={tooltip}>
+            <Box sx={{ height: 24 }}>
+              <StyledTitleTooltipIcon
+                sx={{ marginLeft: 0 }}
+                id="more-info-outlined"
+                width={24}
+                height={24}
+                data-testid={`${rest['data-testid']}-tooltip-icon`}
+              />
+            </Box>
+          </Tooltip>
+        </RenderIf>
       </Box>
       <Autocomplete
-        renderInput={(params) => {
+        renderInput={(params: AutocompleteRenderInputParams) => {
           const { InputLabelProps: _InputLabelProps, ...rest } = params;
 
           return <TextField {...rest} placeholder={placeholder} name={name} />;
         }}
+        sx={{
+          '& .MuiInputBase-root': {
+            borderBottomLeftRadius: shouldShowWarningMessage ? 0 : variables.borderRadius.sm,
+            borderBottomRightRadius: shouldShowWarningMessage ? 0 : variables.borderRadius.sm,
+          },
+        }}
         options={combinedOptions}
-        renderOption={({ children: _children, ...props }, { id, ...psProps }) => (
+        renderOption={(
+          { children: _children, ...props },
+          { id, tag, secretId, nickname, ...psProps },
+        ) => (
           <ParticipantSnippet<'li'>
             key={id}
+            tag={tag}
+            secretId={tag === 'Team' ? nickname : secretId}
+            nickname={tag === 'Team' ? null : nickname}
             {...psProps}
             boxProps={{
-              sx: { p: theme.spacing(0.6, 1.6), cursor: 'pointer' },
+              sx: {
+                p: theme.spacing(0.6, 1.6),
+                cursor: 'pointer',
+                ':last-child': {
+                  borderBottom: `${variables.borderWidth.md} solid ${variables.palette.surface_variant}`,
+                },
+              },
               ...props,
             }}
           />
         )}
-        getOptionLabel={(value) =>
-          `${value.secretId}${value.nickname ? ` (${value.nickname})` : ''}`
-        }
+        isOptionEqualToValue={(option, value) => option.id === value.id}
+        groupBy={(option: ParticipantDropdownOption) => {
+          if (!showGroups) return undefined;
```

Comment:
You should move the `showGroups` validation to satisfy type for `groupBy?` by returning `undefined` instead of `() => undefined`

Reply:
I'm not sure I catch what you mean here. It looks like that's what's already happening?

Reply:
There's a type error for `groupBy?`
```
Type '(option: ParticipantDropdownOption) => string | undefined' is not assignable to type '(option: ParticipantDropdownOption) => string'.
```

I'm suggesting changing this to 
```suggestion
groupBy={
  !showGroups
    ? undefined
    : (option: ParticipantDropdownOption) => {
        if (option.tag !== 'Team') {
          return t('takeNow.modal.dropdown.participantGrouping');
        } else {
          return t('takeNow.modal.dropdown.teamMemberGrouping');
        }
      }
}
```

Reply:
Interestingly I don't get this type error locally (in-editor or tsc), but I see what you mean now 👍

Reply:
I remember seeing that TS error as well; maybe in your case you weren't getting some tsc errors due to that `sx` issue breaking the TS server in VS Code?